sdk/src/main/scala/com/horizen/certificatesubmitter/CertificateSubmitter.scala

```diff
@@ -248,69 +222,6 @@ class CertificateSubmitter(settings: SidechainSettings,
     Await.result(sidechainNodeViewHolderRef ? GetDataFromCurrentView(getStatus), timeoutDuration).asInstanceOf[SubmissionWindowStatus]
   }
 
-  // No MBTRs support, so no sense to specify btrFee different to zero.
-  private def getBtrFee(referencedWithdrawalEpochNumber: Int): Long = 0
-
-  private[certificatesubmitter] def getFtMinAmount(consensusEpochNumber: Int): Long = {
-    ForkManager.getSidechainConsensusEpochFork(consensusEpochNumber).ftMinAmount
-  }
```

Comment:
I think, if we can leave this parts on CertificateSubmitter, because it is the decision independent to the Strategy.
Can be passed as an argument to the proper strategy methods

Reply:
We do not use them in any other places besides strategies